R/noaa_gefs.R

```diff
@@ -0,0 +1,149 @@
+#' NOAA GEFS tables
+#' 
+#' Access NOAA Global Ensemble Forecast System (GEFS) forecast predictions
+#' at NEON sites. The GEFS is NOAA's longest horizon forecast, extending up
+#' to 30 days at present, issued at 0.5 degree spatial resolution. 
+#' EFI downsamples these forecasts at the coordinates of all NEON sites and
+#' provides efficient access to archives of these forecasts in a simple tabular
+#' format for a subset of variables of interest. 
+#' 
+#' WARNING: This combined dataset contains billions of rows. Filtering
+#' to a forecast issued on specific `start_date`s or other subsets before
+#' `collect()`ing data into R is essential. Be patient, especially on slow
+#' network connections, and handle this data with care. See examples.
+#' 
+#' At each site, 31 ensemble member forecasts are provided
+#' at 3 hr intervals for the first 10 days, and 6 hr intervals for up to 30 days
+#' (840 hr) horizon. Forecasts include the following variables:
+#' - TMP - temperature (K)
+#' - RH - Relative humidity (%)
+#' - PRES - Atmospheric pressure (Pa)
+#' - UGRD - U-component of wind speed (m/s)
+#' - VGRD - V-component of wind speed (m/s)
+#' - APCP - Total precipitation in interval (kg/m^2)
```

Comment:
I learned yesterday that kg/m^2 is the same thing as mm of rain.  :upside_down_face: